docs/dev-guide/contributing/documenting.rst

```diff
@@ -40,52 +39,36 @@ must be run as root should be shown using ``sudo``.
 Docs Layout
 -----------
 
-Relative to the root of pulp, the user guide is stored at ``dev/sphinx/user-guide/``
-and the dev guide is stored at ``docs/sphinx/dev-guide/``.
-
+Platform and plugins each have their own Sphinx project which allow docs to be
+built without checking out additional repositories. The Sphinx project for
+platform and each plugin are located in the ``docs`` directory in the top level
+of the repository. For example, the platform Sphinx project
+`is located here <https://github.com/pulp/pulp/tree/master/docs>`_.
 
-Read the Docs
--------------
-
-Pulp's documentation is hosted on `Read the Docs <http://readthedocs.org>`_.
-Links to all current documentation can be found at
-`http://www.pulpproject.org/docs <http://www.pulpproject.org/docs>`_.
+Doc Hosting
+-----------
 
+Pulp's documentation is hosted on `OpenShift <https://www.openshift.com/>`_.
+and is available at `https://docs.pulpproject.org/ <https://docs.pulpproject.org/>`_.
```

Comment:
I'm trying to do everything with `https`. docs.pulpproject.org has SSL configured with a cert to match.